src/view/sheets/pages/PlayerCharacterInventoryTab.svelte

```diff
@@ -1,70 +1,83 @@
-    import prepareObjectTooltip from "../../dataPreparationHelpers/documentTooltips/prepareObjectTooltip.js";
-    import sortItems from "../../../utils/sortItems.js";
-
-    import SearchBar from "../components/SearchBar.svelte";
-
-    async function configureItem(event, id) {
-        event.stopPropagation();
-
-        await actor.configureItem(id);
-    }
-
-    async function createItem(event) {
-        event.stopPropagation();
-
-        await actor.createItem({ name: "New Object", type: "object" });
-    }
-
-    async function deleteItem(event, id) {
-        event.stopPropagation();
-
-        await actor.deleteItem(id);
-    }
-
-    function getObjectMetadata(item) {
-        return null;
-    }
-
-    function groupItemsByType(items) {
-        return items.reduce((categories, item) => {
-            const { objectType } = item.reactive.system;
-
-            categories[objectType] ??= [];
-            categories[objectType].push(item);
-
-            return categories;
-        }, {});
-    }
-
-    const { objectTypeHeadings } = CONFIG.NIMBLE;
-
-    let actor = getContext("actor");
-    let sheet = getContext("application");
-    let searchTerm = $state("");
-
-    let totalInventorySlots = $derived(
-        actor.reactive.system.inventory.totalSlots ?? 0,
-    );
-    let usedInventorySlots = $derived(
-        actor.reactive.system.inventory.usedSlots ?? 0,
-    );
-    let items = $derived(filterItems(actor.reactive, ["object"], searchTerm));
-    let categorizedItems = $derived(groupItemsByType(items));
-
-    // Currency
-    let currency = $derived(actor.reactive?.system?.currency);
-
-    // Settings
-    let flags = $derived(actor.reactive.flags.nimble);
-    let showEmbeddedDocumentImages = $derived(
-        flags?.showEmbeddedDocumentImages ?? true,
-    );
-    let trackInventorySlots = $derived(flags?.trackInventorySlots ?? true);
+import filterItems from '../../dataPreparationHelpers/filterItems.js';
+import { getContext } from 'svelte';
+import localize from '../../../utils/localize.js';
+import { metadata } from '../../../models/chat/common.js';
+import prepareObjectTooltip from '../../dataPreparationHelpers/documentTooltips/prepareObjectTooltip.js';
+import sortItems from '../../../utils/sortItems.js';
+import { RulesManager } from '../../../managers/RulesManager.js';
+
+import SearchBar from '../components/SearchBar.svelte';
+
+async function configureItem(event, id) {
+	event.stopPropagation();
+
+	await actor.configureItem(id);
+}
+
+async function toggleArmor(event, rules, itemId) {
+	event.stopPropagation();
+	let armorRule = rules.getRuleOfType('armorClass');
+	armorRule.disabled = !armorRule.disabled;
+	await rules.updateRule(armorRule.id, armorRule);
+	if (armorRule.disabled) itemsWithDisabledArmor.push(itemId);
+	else {
+		const index = itemsWithDisabledArmor.indexOf(itemId);
+		if (index > -1) itemsWithDisabledArmor.splice(index, 1);
+	}
+}
```

Comment:
This would work better as an actual function on the object. But for now this works.